### ISSUE TITLE
translate-c: handle functions with no prototype

### ISSUE THREAD

**Issue by andrewrk** [ziglang/zig#505]
foo.h:

```c
int foo();
```

```
$ zig translate-c foo.h --verbose
foo.h:1:5: warning: unsupported type: 'FunctionNoProto'
foo.h:1:5: warning: unable to resolve prototype of function 'foo'
```

**Comment by andrewrk** [comment]
I believe this can be emitted into zig AST as an extern function with var args:

```zig
extern fn foo(...) -> int;
```

**Comment by emekoi** [comment]
@andrewrk fixed by e65809160fed05049f8a3cca463b97fb5e51f5d6.
